infra/base-images/base-runner/Dockerfile

```diff
@@ -56,7 +57,8 @@ RUN git clone https://chromium.googlesource.com/chromium/src/tools/code_coverage
     git checkout edba4873b5e8a390e977a64c522db2df18a8b27d && \
     pip3 install wheel && \
     pip3 install -r requirements.txt && \
-    pip3 install MarkupSafe==0.23
+    pip3 install MarkupSafe==0.23 && \
+    pip3 install coverage
```

Comment:
This probably isn't the right location for this install but doesn't really matter.